src/coreComponents/constitutive/relativePermeability/RelpermDriver.cpp

```diff
@@ -52,6 +50,9 @@ RelpermDriver::RelpermDriver( const geos::string & name,
     setInputFlag( InputFlags::OPTIONAL ).
     setApplyDefaultValue( "none" ).
     setDescription( "Baseline file" );
+
+
+  addLogLevel< logInfo::LineSearch >();
```

Comment:
To keep in the scope of this PR, please revert all modifications that are not on SolverBase children.